test/metabase/pulse/render/js_svg_test.clj

```diff
@@ -302,14 +320,3 @@
                                   (json/generate-string {:value value :goal goal})
                                   (json/generate-string settings)))]
       (validate-svg-string :progress svg-string))))
-
-(deftest categorical-waterfall-test
-  (let [rows     [["apples" 2]
-                  ["bananas" 3]]
-        labels   {:left "bob" :right "dobbs"}
-        settings (json/generate-string {})]
-    (testing "It returns bytes"
-      (let [svg-bytes (js-svg/categorical-waterfall rows labels {})]
-        (is (bytes? svg-bytes))))
-    (let [svg-string (.asString ^Value (js/execute-fn-name @context "categorical_waterfall" rows labels settings (json/generate-string (public-settings/application-colors))))]
-      (validate-svg-string :categorical/waterfall svg-string))))
```

Comment:
Removed this because I created an equivalent scenario inside `waterfall-test` earlier.